sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -374,6 +374,16 @@ querySpecification
        windows?)
     ;
 
+hint
+    : '/*+' hintStatement '*/'
+    ;
+
+hintStatement
+    : hintName=identifier
+    | hintName=identifier '(' parameters+=identifier parameters+=identifier ')'
```

Comment:
This rule doesn't support like `SELECT /*+ INDEX(a b c) */ * FROM t`. To support it, it should be:

    | hintName=identifier '(' parameters+=identifier parameters+=identifier* ')'

Is it intentional?